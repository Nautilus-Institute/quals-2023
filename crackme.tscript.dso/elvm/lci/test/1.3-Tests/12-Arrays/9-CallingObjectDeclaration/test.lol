HAI 1.3
	I HAS A foo ITZ A BUKKIT

	HOW IZ foo fun1
		I HAS A bar ITZ 123
		ME HAS A bar
		VISIBLE bar
	IF U SAY SO

	I IZ foo'Z fun1 MKAY
	foo'Z bar R 4.567
	VISIBLE foo'Z bar
KTHXBYE
