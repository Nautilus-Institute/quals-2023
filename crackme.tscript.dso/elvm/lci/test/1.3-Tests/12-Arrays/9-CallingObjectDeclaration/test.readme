This test checks that the calling object reference variable can be used to
declare variables.
