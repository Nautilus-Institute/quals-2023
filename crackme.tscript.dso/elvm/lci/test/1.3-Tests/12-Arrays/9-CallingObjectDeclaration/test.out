123
4.56
